src/lib/Libattr/attr_fn_l.c

```diff
@@ -277,6 +277,44 @@ set_attr_l(attribute *pattr, long value, enum batch_op op)
 	post_attr_set(pattr);
 }
 
+/**
+ * @brief	Attribute setter function for long type values
+ *
+ * @param[in]	pattr	-	pointer to attribute being set
+ * @param[in]	value	-	value to be set
+ * @param[in]	op		-	operation to do
+ *
+ * @return	void
+ *
+ * @par MT-Safe: No
+ * @par Side Effects: None
+ *
+ */
+void
+set_attr_ll(attribute *pattr, Long value, enum batch_op op)
```

Comment:
shouldn't this function be inside attr_fn_ll.c ?

Reply:
yeah, this should be in the attr_fn_ll.c file